### PR TITLE
Refactor the event journaling and update daemon dss-events-scribe

### PR DESCRIPTION
**Pull request by xbrianh**
This refactors the event journaling and update daemon using a more elastic rate-limiting algorithm.

depends on #2654 
connected to #2648

### CODE REVIEW

File: daemons/dss-events-scribe/app.py [hunk 1]
```diff
@@ -28,48 +21,43 @@
 app = domovoi.Domovoi()
 
 
-class ReplicaStatus:
-    prefix = None
-    finished_journaling = False
-    finished_updating = False
+# TODO: Make this configurable
+NUMBER_OF_EVENTS_PER_JOURNAL = 10 if "dev" == os.environ['DSS_DEPLOYMENT_STAGE'] else 1000
```
Comment (DailyDreaming) [inline daemons/dss-events-scribe/app.py]: If `dev` is different, could this potentially cause unexpected behavior when releasing to `integration`?

Comment (amarjandu) [reply daemons/dss-events-scribe/app.py]: I think it has to do with the amount of activity on integration, but i could be wrong.....

Comment (xbrianh) [reply daemons/dss-events-scribe/app.py]: I don't think so. `NUMBER_OF_EVENTS_PER_JOURNAL` is set dynamically at runtime.

File: daemons/dss-events-scribe/app.py [hunk 2]
```diff
@@ -28,48 +21,43 @@
...
 
-    def __init__(self, prefix):
-        self.prefix = prefix
 
+@app.sqs_queue_subscriber("dss-events-scribe-" + os.environ['DSS_DEPLOYMENT_STAGE'],
+                          batch_size=1,
+                          queue_attributes=dict(VisibilityTimeout="600"))
+def handle_sqs_message(event, context):
+    if 1 != len(event['Records']):
+        logger.error(f"Received {len(event['Records'])}, expected 1")
```
Comment (amarjandu) [inline daemons/dss-events-scribe/app.py]: if the batch_size changes will there be an error? Might want to throw in a comment to that these values can be changed from the lambda console, under lambda triggers

Comment (xbrianh) [reply daemons/dss-events-scribe/app.py]: This check ensures there is an error if `batch_size != 1`. I'll add a comment.

**Pull request merged**